sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/AzureMonitorExporterOptions.cs

```diff
@@ -1,12 +1,17 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using System.ComponentModel;
+
 using Azure.Core;
 
 namespace Azure.Monitor.OpenTelemetry.Exporter
 {
     public class AzureMonitorExporterOptions : ClientOptions
```

Comment:
Options should define a ctor that takes ServiceVersion as a defaulted parameter: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationClientOptions.cs#L47